ipcs_domain/src/main/java/com/ipcs/service/impl/SecurityServiceImpl.java

```diff
@@ -61,6 +61,15 @@ public void setRoleDao(RoleDao roleDao) {
         this.roleDao = roleDao;
     }
 
+
+    @Transactional
+    public long authenticateLoginInfoWithID(String userName, String password) {

```

Comment:
method authenticateLoginInfoWithID name is misleading.
Please change name, like getAuthenticatedUserID